CIP-0001/README.md

```diff
@@ -1,185 +1,374 @@
+
 <details>
   <summary><span id="licensing"><strong>Licensing</strong></span></summary>
 
-The following licenses are accepted. Each new CIP must identify at least one acceptable license in its preamble. In addition, each license must be referenced by its respective abbreviation below.
+CIPs are licensed in the public domain. Moreso, they must be licensed under one of the following licenses. Each new CIP must identify at least one acceptable license in its preamble. In addition, each license must be referenced by its respective abbreviation below in the _"Copyright"_ section.
 
 #### Recommended licenses
 
-- for software / code: Apache-2.0: [Apache License, version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
-- for documentation: CC-BY-4.0: [Creative Commons Attribution 4.0 International Public License](https://creativecommons.org/licenses/by/4.0/legalcode)
+- for software / code: Apache-2.0 - [Apache License, version 2.0][Apache-2.0]
+- for documentation: CC-BY-4.0 - [Creative Commons Attribution 4.0 International Public License][CC-BY-4.0]
 
 #### Unacceptable licenses
 
 All licenses not explicitly included in the above lists are not acceptable terms for a Cardano Improvement Proposal unless a later CIP extends this one to add them.
 
 </details>
 
-<p align="right">
-  A reference template is available in <a href="https://github.com/cardano-foundation/CIPs/tree/master/.github/CIP-TEMPLATE.md">.github/CIP-TEMPLATE.md</a>.
-</p>
+> **Note** A reference template is available in [.github/CIP-TEMPLATE.md][CIP-TEMPLATE.md]
+
+#### Statuses
+
+CIPs can have three statuses: `Proposed`, `Active` or `Inactive`. [The CIP Process section](#the-cip-process) highlights how CIPs move through these statuses; no CIP should be given one of these statuses without satisfying the criteria described here below.
+
+> **Note** There is no "draft" status: a proposal which has not been merged (and hence exists in a PR) is a draft CIP. Draft CIPs should include the status they are aiming for on acceptance. Typically, but not always, this will be _'Proposed'_.
+
+##### Status: Proposed
+
+A _'Proposed'_ CIP is any CIP that meets the essential CIP criteria but is not yet _'Active'_. The criteria that must meet a CIP to be merged as _'Proposed'_ are:
+
+- It must contain all the sections described in [Structure](#structure).
+- The quality of the content must be to the Editors’ satisfaction. That means it must be grammatically sound, well-articulated and demonstrates noticeable efforts in terms of completeness and level of detail.
+- Its technical soundness should have been established. Where necessary, this may require review by particular experts and addressing their concerns. Note that the requirement is that the proposal makes sense (i.e. be technically sound), yet no consulted experts need to think it is a good idea.
+- It must have a valid _'Path to Active'_. This particular section must be subdivided into two sub-sections:
+
+  - _'Acceptance Criteria'_
```

Comment:
Pondering this in the case of Plutus builtins in particular, and I was wondering about insisting that CIPs proposing Plutus builtins be required to include test cases, or more generally a test plan. Because often the proposer is the one who actually knows what it's supposed to do, so they should include them here.

That made me wonder whether it's something that we want to include in _any_ CIP: the acceptance criteria SHOULD include test cases that must pass, and the implementation plan SHOULD include a testing plan.

WDYT?

Reply:
I guess more generally I'm thinking that Acceptance Criteria usually involve some expectations of behaviour change, and it's not unreasonable to expect the proposer to at least broadly sketch out the expected behaviour changes.

e.g. for reference inputs we could have asked me to provide some transaction submission scenarios that would now work. That then is very helpful for generating test cases.